cmd/collectors/rest/rest.go

```diff
@@ -295,7 +295,16 @@ func (r *Rest) PollData() (*matrix.Matrix, error) {
 		for label, display := range r.instanceLabels {
 			value := instanceData.Get(label)
 			if value.Exists() {
-				instance.SetLabel(display, value.String())
+				if value.IsArray() {
+					var labelArray []string
+					for _, r := range value.Array() {
+						labelString := r.String()
+						labelArray = append(labelArray, labelString)
+					}
+					instance.SetLabel(display, strings.Join(labelArray[:], ","))
```

Comment:
minor but no need to copy labelArray before calling Join

Reply:
where is the copy?

Reply:
corrected.